### PR TITLE
Add mainnet resources

### PR DESCRIPTION
**Pull request by anilcse**

### CODE REVIEW

File: regen-1/README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
-## Requirements
+## Resources
+
+- ### [Block Explorer](./explorer-urls.txt)
+- ### [Wallets](./wallets.txt)
+- ### [API Nodes](./api-nodes.txt)
+- ### [RPC Nodes](./rpc-nodes.txt)
+- ### [Seed Nodes](./seed-nodes.txt)
+- ### [Persistent Peer Nodes](./peer-nodes.txt)
+- ### [API Nodes](./api-nodes.txt)
```
Comment (clevinson) [inline regen-1/README.md]:
```suggestion
```

This line is a duplicate

**Review by clevinson — CHANGES_REQUESTED** [review]
Nice additions, thanks @anilCSE !

Few thoughts:
- Can we add gRPC API links as well? Do we know of any nodes running a public grpc api?
- We should remove the `regen-prelaunch-3` task from `Makefile`
- Can we link somewhere (idk if it belongs in `api-nodes.txt` or maybe just as alink in `regen-1/README.md` to an instance of the live swagger docs? Not all API nodes will have swagger enabled, so it'd be useful to link out to one if we know of a public endpoint.

**Comment by anilcse** [comment]
> Nice additions, thanks @anilCSE !
> 
> Few thoughts:
> 
> * Can we add gRPC API links as well? Do we know of any nodes running a public grpc api?
> * We should remove the `regen-prelaunch-3` task from `Makefile`
> * Can we link somewhere (idk if it belongs in `api-nodes.txt` or maybe just as alink in `regen-1/README.md` to an instance of the live swagger docs? Not all API nodes will have swagger enabled, so it'd be useful to link out to one if we know of a public endpoint.

Thanks @clevinson , updated the PR with relavant info

File: Makefile [hunk 2]
```diff
@@ -2,9 +2,6 @@ all: prelaunch-3 regen-1
```
Comment (clevinson) [inline Makefile:2]: We should also drop `prelaunch-3` from line 1, and line 3.

Comment (anilcse) [reply Makefile:2]: My bad :facepalm: , updated now.

**Review by clevinson — APPROVED** [review]

**Pull request merged**